### ISSUE TITLE
'High Contrast Black' theme Input Group Error State WCAG AAA contrast compliance

### ISSUE THREAD

**Issue by prsdthkr** [SAP/fundamental-styles#1006]
**Describe the bug**
The  Input Group components's Error Input color contrast ratio is not 7:1 or above.

> What version of the Fundamental Library Styles are you using?
v0.9.1-rc.0

> What front-end framework are you implementing in, e.g., Angular, React, etc.?
React

**To Reproduce**
1. Go to https://deploy-preview-928--fundamental-styles.netlify.app/components/input-group.html
2. Check the Error input color contrast

**Expected behavior**
If we want WCAG AAA compliance, this contrast ratio should be 7:1 or above.

**Screenshots**
![image](https://user-images.githubusercontent.com/43764832/81747068-5740fb80-945c-11ea-97a0-81c895f74d3e.png)
![image](https://user-images.githubusercontent.com/43764832/81747119-6758db00-945c-11ea-9079-11d604118277.png)

**Desktop (please complete the following information):**
 - OS: macOS
 - Browser: Chrome 
 - Version: 81.0.4044.129 (Official Build) (64-bit)
